### PR TITLE
Update to version 0.5

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by jakirkham** [comment]
Needs to be [re-rendered]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with `conda-smithy` version `1.5.3`.

**Comment by kynan** [comment]
@jakirkham done!

**Comment by kynan** [comment]
@jakirkham Is this good to go?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 package:
   name: backports_abc
   version: {{ version }}
 
 source:
-  fn: backports_abc-0.4.tar.gz
+  fn: backports_abc-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/b/backports_abc/backports_abc-{{ version }}.tar.gz
-  md5: 0b65a216ce9dc9c1a7e20a729dd7c05b
+  md5: 7d1936ec183a3586290adf60f6f96764
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we please switch to `sha256`?

**Comment by jakirkham** [comment]
Basically, would switch to `sha256` as provided by [pypi.org]( https://pypi.org ). Otherwise LGTM.

File: recipe/meta.yaml (hunk 1 above)
Comment (kynan) [reply recipe/meta.yaml]: Done!

**Comment by jakirkham** [comment]
Thanks @kynan .

**Comment by jakirkham** [comment]
Should add feel free to add yourself as a maintainer if you are interested.

**Pull request merged**